baggage/baggage.go

```diff
@@ -36,32 +36,32 @@ func Baggage(ctx context.Context) label.Set {
 	return label.NewSet(values...)
 }
 
-// BaggageValue returns the value related to key in the baggage of ctx. If no
+// Value returns the value related to key in the baggage of ctx. If no
 // value is set, the returned label.Value will be an uninitialized zero-value
 // with type INVALID.
-func BaggageValue(ctx context.Context, key label.Key) label.Value {
+func Value(ctx context.Context, key label.Key) label.Value {
 	v, _ := baggage.MapFromContext(ctx).Value(key)
 	return v
 }
 
-// ContextWithBaggageValues returns a copy of parent with pairs updated in the baggage.
-func ContextWithBaggageValues(parent context.Context, pairs ...label.KeyValue) context.Context {
+// ContextWithValues returns a copy of parent with pairs updated in the baggage.
+func ContextWithValues(parent context.Context, pairs ...label.KeyValue) context.Context {
 	m := baggage.MapFromContext(parent).Apply(baggage.MapUpdate{
 		MultiKV: pairs,
 	})
 	return baggage.ContextWithMap(parent, m)
 }
 
-// ContextWithoutBaggageValues returns a copy of parent in which the values related
+// ContextWithoutValues returns a copy of parent in which the values related
 // to keys have been removed from the baggage.
-func ContextWithoutBaggageValues(parent context.Context, keys ...label.Key) context.Context {
+func ContextWithoutValues(parent context.Context, keys ...label.Key) context.Context {
```

Comment:
nit: `ctx` instead of `parent`
```suggestion
func ContextWithoutValues(ctx context.Context, keys ...label.Key) context.Context {
        m := baggage.MapFromContext(ctx).Apply(baggage.MapUpdate{
 		DropMultiK: keys,
 	})
 	return baggage.ContextWithMap(ctx, m)
 }
```